pkg/querier/blocks_store_queryable.go

```diff
@@ -687,14 +691,18 @@ func (q *blocksStoreQuerier) fetchSeriesFromStores(
 
 			numSeries := len(mySeries)
 			chunkBytes := countChunkBytes(mySeries...)
+			labelBytes := countDataBytes(mySeries...)
+			dataBytes := labelBytes + chunkBytes
```

Comment:
So here `labelBytes := countDataBytes(mySeries...)` already includes both chunk bytes and label bytes so we count `chunkBytes` twice.